openmls/src/group/mls_group/creation.rs

```diff
@@ -20,13 +20,17 @@ impl MlsGroup {
     pub fn new(
         backend: &impl OpenMlsCryptoProvider,
         mls_group_config: &MlsGroupConfig,
+        life_time: LifetimeExtension,
```

Comment:
This is an API choice in the end, so @raphaelrobert probably has an opinion, but since we're creating a new group, we can just go with the default. If I understand correctly, the extension is just used in the founder's initial leaf KeyPackage, which is not going to expire anyway.

Reply:
I think the lifetime should be part of the `MlsGroupConfig`.